src/jit/compiler.h

```diff
@@ -8263,7 +8263,14 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         var_types compRetNativeType; // Normalized return type as per target arch ABI
         unsigned  compILargsCount;   // Number of arguments (incl. implicit but not hidden)
         unsigned  compArgsCount;     // Number of arguments (incl. implicit and     hidden)
-        unsigned  compRetBuffArg;    // position of hidden return param var (0, 1) (BAD_VAR_NUM means not present);
+
+#if FEATURE_FASTTAILCALL
+        unsigned compArgRegCount;      // Number of incoming integer args
+        unsigned compFloatArgRegCount; // Number of incoming floating point args
+        size_t   compStackSize;        // Incoming stack size
```

Comment:
The comments here say `Number of incoming XX args`. But presumably they should be more precise and say, `Number of incoming XX argument registers used for incoming arguments`.